packages/connect-common/scripts/check-all-firmware-revisions.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+PARENT_PATH=$( cd "$(dirname "${BASH_SOURCE[0]}")" || exit ; pwd -P )
+
+cd "$PARENT_PATH/../files/firmware" || exit
+
+for dir in */ ; do
+    if [ -d "$dir" ]; then
+        "$PARENT_PATH/check-firmware-revisions.sh" "${dir%/}"
```

Comment:
I think this wont propagate the bash return code. If you run this with some error will it produce and `1` when doing `echo $?`?

Reply:
I think you have to explicitly check for it like this https://github.com/trezor/data/pull/90/files#diff-174612db9a6f23fe0db7575ea6471cddf33039da8341189af284c0084d0ee632R8